releasenotes/notes/Parameterized-commutation-checker-8a78a4715bf78b4e.yaml

```diff
@@ -0,0 +1,9 @@
+---
+features_circuits:
+  - |
+    Improved the functionality of :class:`.CommutationChecker` to include support for
```

Comment:
Looking at this, `CommutationChecker` is actually not API documented. It would probably be better to reference an API-documented class like `CommutationAnalysis` and state that it now correctly finds commutations of the parameterized gates.

Reply:
I think this is a docs bug, it was there in 1.0: https://docs.quantum.ibm.com/api/qiskit/1.0/qiskit.circuit.CommutationChecker I think it got lost from the docs in @jakelishman's big refactor of the circuit module docs in #11904

Reply:
I was surprised at that, because I had tried to made a big list of every documented thing and was crossing them off as I did them.  Turns out that it wasn't documented before, and the docs for it got added in the release-notes prep PR for 1.0.0 (#11760), but that change doesn't seem to have got backported to `main`.

Reply:
Ah in that case @MarcDrudis you can leave the link to `CommutationChecker` and we should add the docs link.

Reply:
I made a small PR to add the `CommutationChecker` to the docs: https://github.com/Qiskit/qiskit/pull/13303 feel free to edit/delete if the style or info doesn't fit 🙂